### PR TITLE
Use current backlog page instead of old (fixes #5719)

### LINKED ISSUE #5719: Opening rcon resets current page in console
When scrolling up and reading a conversation in the console the current page is saved. You can open and close the console and the page is still there.
If you open rcon (f2) and switch back to the console the current page is resetted to 1 or 2. Thats pretty annoying and the current page in the console should stay the same.
Could be the same issue the other way around but i didnt test that.

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/game/client/components/console.cpp [hunk 1]
```diff
@@ -763,7 +763,7 @@ void CGameConsole::OnRender()
 		if(pConsole->m_BacklogCurPage == INT_MAX)
 			s_LastActivePage = INT_MAX;
 		int TotalPages = 1;
-		for(int Page = 0; Page <= s_LastActivePage; ++Page, OffsetY = 0.0f)
+		for(int Page = 0; Page <= pConsole->m_BacklogCurPage; ++Page, OffsetY = 0.0f)
```
Comment (Robyt3) [inline src/game/client/components/console.cpp:766]:
```suggestion
		for(int Page = 0; Page <= maximum(s_LastActivePage, pConsole->m_BacklogCurPage); ++Page, OffsetY = 0.0f)
```

File: src/game/client/components/console.cpp [hunk 2]
```diff
@@ -763,7 +763,7 @@ void CGameConsole::OnRender()
 		if(pConsole->m_BacklogCurPage == INT_MAX)
 			s_LastActivePage = INT_MAX;
```
Comment (Robyt3) [inline src/game/client/components/console.cpp:764]:
```suggestion
```

**Comment by Robyt3** [comment]
Your change fixes the issue, but they make the console contents flash for a frame when scrolling down.

The changes I suggested above should fix the issue without making the contents flash on scroll. They also seem to fix the existing issue that the console contents flash when you hold down the Home key.

**Comment by def-** [comment]
Thanks, even better! Can confirm the original issue is still fixed and no more flashing

**Pull request merged**